modules/executors/variables.tf

```diff
@@ -76,7 +76,7 @@ variable "queue_name" {
 
 variable "maximum_runtime_per_job" {
   type        = string
-  default     = "30m"
+  default     = "45m"
```

Comment:
this is the default we use on managed instances and dotcom - proven to work well.